Library/Tracking/KSRAnalytics.swift

```diff
@@ -1806,7 +1815,7 @@ private func contextProperties(
 
   result["cta"] = ctaContext?.trackingString
   result["location"] = locationContext?.trackingString
-  result["page"] = page?.rawValue
+  result["page"] = page?.rawValue ?? "other"
```

Comment:
leave an inline comment here explaining why we're using "other" based on insights definition.